daisy/README.md

```diff
@@ -25,6 +25,15 @@ Other use-case examples:
 Daisy documentation can be found
 [here](https://googlecloudplatform.github.io/compute-image-tools/daisy.html).
 
+# Logging
+
+Daisy will send logs to Cloud Logging. If the API is disabled or the
```

Comment:
It would be good to make "Cloud Logging" a link to the docs page for Cloud Logging (https://cloud.google.com/logging/), so people can quickly find out what it is, especially since we often call it "Stackdriver Logging" externally.

Reply:
Also, consider adding a note about the logging flags here: https://googlecloudplatform.github.io/compute-image-tools/daisy-installation-usage.html